src/scripts/main.js

```diff
@@ -1,3 +1,65 @@
 'use strict';
 
-// write code here
+const removeColumn = document.querySelector('.remove-column');
+const appendColumn = document.querySelector('.append-column');
+const removeRow = document.querySelector('.remove-row');
+const appendRow = document.querySelector('.append-row');
+const field = document.querySelector('tbody');
+const max = 10;
+const min = 2;
```

Comment:
constants are uppercased
```suggestion
const MAX = 10;
const MIN = 2;
```